src/main/java/org/kainos/ea/db/SalesDAO.java

```diff
@@ -0,0 +1,50 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.SalesEmployee;
+import org.kainos.ea.cli.SalesEmployeeRequest;
+import org.kainos.ea.client.FailedToUpdateSalesEmployeeException;
+import org.kainos.ea.client.SalesEmployeeDoesNotExistException;
+
+import java.sql.*;
+
+public class SalesDAO {
+    private DbConnector databaseConnector = new DbConnector();
+
+    public int createSalesEmployee(SalesEmployeeRequest salesEmployee) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO Sales_Employee (Name, Salary, bank_number, ni, commission_rate) VALUES (?,?,?,?,?)";
```

Comment:
do you not have an employee table that also needs updated?

Reply:
No